bpf/init.sh

```diff
@@ -19,26 +19,27 @@ RUNDIR=$2
 IP4_HOST=$3
 IP6_HOST=$4
 MODE=$5
-# Only set if MODE = "direct", "ipvlan", "flannel"
-NATIVE_DEVS=$6
-HOST_DEV1=$7
-HOST_DEV2=$8
-XDP_DEV=$9
-XDP_MODE=${10}
-MTU=${11}
-IPSEC=${12}
-ENCRYPT_DEV=${13}
-HOSTLB=${14}
-HOSTLB_UDP=${15}
-HOSTLB_PEER=${16}
-CGROUP_ROOT=${17}
-BPFFS_ROOT=${18}
-NODE_PORT=${19}
-NODE_PORT_BIND=${20}
-MCPU=${21}
-NR_CPUS=${22}
-ENDPOINT_ROUTES=${23}
-PROXY_RULE=${24}
+# Set if MODE = "direct", "ipvlan", "flannel", "tunnel"
```

Comment:
AFAIU, this comment is for `NATIVE_DEVS`, not for `TUNNEL_MODE`. We might need a similar comment for `TUNNEL_MODE` though.

Reply:
Fixed. Thanks for the feedback!